### ISSUE TITLE
Can't build electron application for dev

### ISSUE THREAD

**Issue by alexdevero** [laurent22/joplin#269]
When I try to build the electron application, following the instructions in build.md, It always fails with error that "rsync is not recognized..." on `rsync --delete -a ../../ReactNativeClient/lib/ lib/`.

### Operating system

- [X] Windows
- [ ] macOS
- [ ] Linux
- [ ] Android
- [ ] iOS

### Application

- [X] Desktop
- [ ] Mobile
- [ ] Terminal

**Comment by laurent22** [comment]
On Windows, the instructions are actually in https://github.com/laurent22/joplin/blob/master/appveyor.yml

You will need yarn and node. Then it's:

```
cd Tools
npm install
cd ..\ElectronClient\app
xcopy /C /I /H /R /Y /S ..\..\ReactNativeClient\lib lib
npm install
yarn dist
```

**Comment by alexdevero** [comment]
Thanks!

**Comment by alexdevero** [comment]
Is it possible to run Joplin on a "dev server" - seeing changes in code without building the app over and over again?

**Comment by alexdevero** [comment]
Maybe, `the package.json` could have a "start" script with `electron .` command.

**Comment by laurent22** [comment]
Yes something like this would make sense. It would also need to update the lib and run `npm run compile` (to convert the JSX to JS) as is done in https://github.com/laurent22/joplin/blob/master/ElectronClient/build.sh

**Comment by alexdevero** [comment]
So, something like `npm run compile && electron .`?

**Comment by laurent22** [comment]
More like `rsync -a --delete "../../ReactNativeClient/lib/" "lib/" && npm run compile && electron .`

So far I've been using the script build.sh to build the app and run.sh to run it, but indeed these could be simplified and made more standard, for example by creating a node script for copying the lib, and another that combines copying the lib, compile the JSX and run the app.

**Comment by alexdevero** [comment]
Yes, you are right, Also, `rsync` didn't work for me lately.

**Comment by laurent22** [comment]
I guess this is now addressed in #272
